content/blog/cloudnative-pg-1-23.0-released/index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "CloudNativePG 1.23.0, 1.22.3 and 1.21.5 Released!"
+date: 2024-04-25T16:04:04+02:00
+draft: false
+image:
+    url: wildlife-zoo-mammal-fauna-elephant-sri-lanka-988769-pxhere.com.jpg
+    attribution: from <strong><a href="https://pxhere.com/en/photo/988769?utm_content=clipUser&utm_medium=referral&utm_source=pxhere">PxHere</a></strong>
+author: gbartolini
+tags:
+ - release
+ - postgresql
+ - postgres
+ - kubernetes
+ - k8s
+ - cloudnativepg
+ - pdb
+ - imagecatalog
+summary: The CloudNativePG community has released the new 1.23 minor version and a new update for the supported 1.22 and 1.21 versions of the CloudNativePG operator.
+---
+The **CloudNativePG Community** is excited to announce the release of version
+1.23.0 of the **CloudNativePG Operator**!
+
+This release brings a host of new features and enhancements, including support
+for PostgreSQL image catalogs, synchronization of user-defined replication
+slots, and Pod Disruption Budget (PDB) configuration.
+
+We've revised our Community support policy to prioritize one minor release at a
+time, enhancing our focus and resources. Furthermore, we've extended the
+supplementary support period for the prior minor release to three months.
```

Comment:
This makes it sound like we only have one release?
My understanding was we were going from 3 to 2?

Reply:
Effectively, this is what is going to happen. We went from 2 minor releases + 1 additional month, to 1 minor release + 3 months. So for at least 3 months we need to support 2 minor releases. With a 3 month delivery time of a release (like now), it still brings to a total of roughly 6 months of support.